app/controllers/provider_interface/organisation_permissions_controller.rb

```diff
@@ -78,10 +82,30 @@ def organisation_id_and_permission_check
       render_403 unless relationship_providers.map { |p| auth.can_manage_organisation?(provider: p) }.any?
     end
 
+    def provider_relationships_to_display
+      if FeatureFlag.active?(:account_and_org_settings_changes)
+        ProviderRelationshipPermissions.includes(:training_provider, :ratifying_provider).all_relationships_for_providers([provider]).providers_have_open_course
+      else
+        ProviderRelationshipPermissions.all_relationships_for_providers([provider]).where.not(setup_at: nil)
+      end
+    end
+
     def sort_relationships_by_provider_name(relationships, provider)
       relationships.sort_by do |relationship|
         relationship.training_provider == provider ? relationship.ratifying_provider.name : relationship.training_provider.name
       end
     end
+
+    def change_link_for_relationship(relationship)
```

Comment:
can this method move to the component where we can test it?

Reply:
Yes we can.

The slight complication is that we do need the change link href to be configurable for the different flows this is used in. But it's definitely possible to clean this up by putting it in the component